Content.Server/Ghost/GhostSystem.cs

```diff
@@ -313,6 +318,17 @@ private void OnGhostWarpToTargetRequest(GhostWarpToTargetRequestEvent msg, Entit
                 return;
             }
 
+            // Frontier: check admin status when warping to admin-only warp points
+            if (_admin.IsAdmin(attached) &&
+                TryComp<WarpPointComponent>(target, out var warpPoint) &&
+                warpPoint.AdminOnly)
+            {
+                Log.Warning($"User {args.SenderSession.Name} tried to warp to an admin-only warp point: {msg.Target}");
+                _adminLogger.Add(LogType.Action, LogImpact.Medium, $"{EntityManager.ToPrettyString(attached):player} tried to warp to admin warp point {EntityManager.ToPrettyString(msg.Target)}");
+                return;
```

Comment:
Wont this allow spamming the logs a bit too much with auto clicker?

Reply:
The button for this doesn't appear on UI (unless you were an admin, opened the panel, closed it, disabled admin and were near frame-perfect on clicking it) - truthfully, I don't even know how you'd do this, but it's something that should be logged - something is wrong here.

Reply:
Based